package.json

```diff
@@ -91,7 +91,7 @@
     "@angular/platform-server": "^12.2.8",
     "@angular/router": "^12.2.8",
     "@babel/cli": "^7.12.10",
-    "@commitlint/cli": "^15.0.0",
+    "@commitlint/cli": "^14.1.0",
```

Comment:
Please make sure you are not changing the package or package lock.json. I don't think that you need to change any dependencies for this fix.